code/datums/martial/monk.dm

```diff
@@ -0,0 +1,455 @@
+			ki_level = 50
+		if(5)
+			to_chat(owner, "<span class = 'danger'>You feel more confident in your strikes, and your body is pure.</span>")
+			flurry_of_blows_penalty = -1
+			purity_of_body = TRUE
+		if(7)
+			to_chat(owner, "<span class = 'danger'>You can perform more Stunning Fists before resting. You have figured out the wholeness of your body.</span>")
+			stunning_fist.max_uses = 7
+			wholeness_of_body.Grant(owner)
+			wholeness_of_body.martial = src
+			available_actions = list(stunning_fist, wholeness_of_body)
+		if(8)
+			to_chat(owner, "<span class = 'danger'>You move from one downed target to the next quickly.</span>")
+			cleave_level = 1
+		if(9)
+			to_chat(owner, "<span class = 'danger'>You have become completely confident in your ability to land a strike.</span>")
+			flurry_of_blows_penalty = 0
+		if(10)
+			to_chat(owner, "<span class = 'danger'>Your Ki helps you seek the holes in enemy armor.</span>")
+			ki_level = 75
+		if(11)
+			to_chat(owner, "<span class = 'danger'>You feel confident enough to strike more and harder, and use this against toxins in your body.</span>")
+			attacks_w_flurry = 3
+			diamond_body = TRUE
+		if(12)
+			to_chat(owner, "<span class = 'danger'>You now use a circle kick to extend your attacks to other foes in combat.</span>")
+			circle_kick = TRUE
+		if(14)
+			to_chat(owner, "<span class = 'danger'>You use a downed foe to strike at their allies.</span>")
+			cleave_level = 2
+		if(15)
+			to_chat(owner, "<span class = 'danger'>You have practiced the Quivering Palm technique enough to use it in combat.</span>")
+			quivering_palm.Grant(owner)
+			quivering_palm.martial = src
+			available_actions = list(stunning_fist, wholeness_of_body, quivering_palm)
+		if(16)
+			to_chat(owner, "<span class = 'danger'>Your Ki has fully developed, and can be used to nullify armor.</span>")
+			ki_level = 100
+		if(17)
+			to_chat(owner, "<span class = 'danger'>You can perform more Stunning Fists before resting.</span>")
+			stunning_fist.max_uses = 14
+		if(20)
+			if(!is_disciple)
+				to_chat(owner, "<span class = 'danger'>You can now make your first Disciple.</span>")
+				make_disciple.Grant(owner)
+				make_disciple.uses++
+		if(25)
+			if(!is_disciple)
+				to_chat(owner, "<span class = 'danger'>You can now make your second Disciple.</span>")
+				make_disciple.uses++
+		if(30)
+			if(!is_disciple)
+				to_chat(owner, "<span class = 'danger'>You can now make your final Disciple.</span>")
+				make_disciple.uses++
+
+/datum/martial_art/monk/proc/attack_roll(mob/living/T, abm)
+	if(istype(T, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = T
+		if(H.mind && istype(H.mind.martial_art, /datum/martial_art/monk))
+			var/datum/martial_art/monk/M = H.mind.martial_art
+			return M.defense_roll(curr_ab + abm)
+	var/armor_class = 10 + (T.getarmor("chest", "melee") * 0.1)
+	var/attack_bonus = curr_ab + abm
+	if(T.incapacitated())
+		attack_bonus += 3

```

Comment:
May want to swap this to using proper Coup De Grace rules, but that might be a little extreme.

> A full-round action that allows an attacker to attempt a killing blow against a helpless opponent. A coup de grace can be administered with a melee weapon, or with a bow or crossbow if the attacker is adjacent to the opponent. An attacker delivering a coup de grace automatically scores a critical hit, after which the defender must make a Fortitude save (DC 10 + damage dealt) or die. Rogues also gain their extra sneak attack damage for this attack. Delivering a coup de grace provokes attacks of opportunity from threatening foes. A coup de grace is not possible against a creature immune to critical hits.

Reply:
I mean if you are in a state of complete helplessness and the attack takes a reasonable amount of time, nothing really changes here. You could just as easilly kill them with kicks. Sleeping Carp has Elbow Drop, which is basically a guaranteed Coup De Grace.

Reply:
Reminder but if I remember right a full *round* in dnd is 3 seconds.

Reply:
6 seconds.